docs/tables/aws_elasticache_cluster_log_delivery_configurations.md

```diff
@@ -0,0 +1,14 @@
+
+# Table: aws_elasticache_cluster_log_delivery_configurations
+Returns the destination, format and type of the logs.
+## Columns
+| Name        | Type           | Description  |
+| ------------- | ------------- | -----  |
+|cluster_cq_id|uuid|Unique CloudQuery ID of aws_elasticache_clusters table (FK)|
+|destination_details_cloud_watch_logs_details_log_group|text|The name of the CloudWatch Logs log group.|
```

Comment:
Bit of a nit, but it might be nice to shorten this column name to remove the 2x `details`. Maybe `destination_cloud_watch_logs_log_group`?